cmd/webhook/main.go

```diff
@@ -78,6 +79,13 @@ var (
 	// https://github.com/sigstore/policy-controller/issues/354
 	disableTUF = flag.Bool("disable-tuf", false, "Disable TUF support.")
 
+	// Validate specific resources.
+    // https://github.com/sigstore/policy-controller/issues/1388
```

Comment:
nit, but the format looks off here?

Reply:
seems something related to git.
I'm using intelji and format looks good.
![image](https://github.com/user-attachments/assets/0f4d7976-1cf1-44bc-8494-ecb3f6da46a9)

Reply:
Ran the command: gofmt -s -w .